plugins/proofpoint_tap/plugin.spec.yaml

```diff
@@ -5,7 +5,7 @@ name: proofpoint_tap
 title: Proofpoint TAP
 description: Parse Proofpoint Targeted Attack Protection (TAP) alerts
 version: 3.1.2
-supported_versions: ["Proofpoint TAP API v2", "Tested on 2021-8-30"]
+supported_versions: ["Proofpoint TAP API v2 tested on 2021-08-30"]
```

Comment:
@mberezin-r7 I think original reads better, effectively saying: You need v2 API and it was last tested on this date.

Reply:
Going to replace this PR with https://github.com/rapid7/insightconnect-plugins/pull/1007